pkg/replica/backup.go

```diff
@@ -202,11 +202,11 @@ func (rb *BackupStatus) CompareSnapshot(snapID, compareSnapID, volumeID string)
 		if val <= byte(from) && val > byte(to) {
 			offset := int64(i) * rb.replica.volume.sectorSize
 			// align
-			offset -= (offset % backupBlockSize)
+			offset -= offset % blockSize
```

Comment:
nit:
I'm fine with `offset -= (offset % backupBlockSize)` because it has better readibility although the `%` operator has higher precedence than `-=`.